src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandArtifact.java

```diff
@@ -20,9 +20,13 @@
 import static org.broadinstitute.hellbender.tools.walkers.annotator.StrandArtifact.StrandArtifactZ.*;
 
 /**
- * Annotations for strand artifact filter
+ * Annotations for strand artifact filter calculated by a Bayesian model described in <a href='https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf' target='_blank'>.
```

Comment:
Need </a> element to surround the text that you want hyperlinked. E.g. 

`calculated by a Bayesian model described in <a href='https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf' target='_blank'>https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf</a>.` shows up as 

calculated by a Bayesian model described in <a href='https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf' target='_blank'>https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf</a>.

Currently we have:
![screenshot 2017-12-11 15 19 08](https://user-images.githubusercontent.com/11543866/33852013-e0511f38-de86-11e7-8b29-eb8acec2df87.png)

Reply:
done